BatteryConditions.cpp

```diff
@@ -0,0 +1,53 @@
+#include "BatteryConditions.h"
+#include "iostream"
+
+
+
+bool BatteryConditions::checkRange(float actualValue, float minValue, float maxValue)

```

Comment:
Currently the method does not indicate if it returns a true value for in range or out of range values. Method name can be more intuitive. for ex. IsInRange(..).